concordium-consensus/Setup.hs

```diff
@@ -41,8 +39,6 @@ makeRust args flags = do
 
 main =
     defaultMainWithHooks $
-        generatingProtos
-            "../concordium-grpc-api/v2"
-            simpleUserHooks
-                { preConf = makeRust
-                }
```

Comment:
How come we do not need `generatingProtos`from base here, but we do in the client?

Reply:
It's because in the client we are **also** generating V1 api until you've made the changes there.

After the client is moved to V2 API completely we will not need it anymore.